stapler/stapler.go

```diff
@@ -0,0 +1,529 @@
+// package stapler implements OCSP stapling feature described here: http://en.wikipedia.org/wiki/OCSP_stapling
+// stapler provides implementation that caches the staple on the first request and periodically updates the stapler
+package stapler
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"github.com/mailgun/vulcand/Godeps/_workspace/src/golang.org/x/crypto/ocsp"
```

Comment:
Minor style issue, can this commit be moved down so it's grouped with the rest of the Godeps.